src/User32/User32.cs

```diff
@@ -246,6 +246,16 @@ public static partial class User32
         [DllImport(nameof(User32), SetLastError = true)]
         public static extern int GetWindowThreadProcessId(IntPtr hWnd, out int lpdwProcessId);
 
+        /// <summary>
+        /// Attaches or detaches the input processing mechanism of one thread to that of another thread.
+        /// </summary>
+        /// <param name="idAttach">The identifier of the thread to be attached to another thread.The thread to be attached cannot be a system thread.</param>
+        /// <param name="idAttachTo">The identifier of the thread to which idAttach will be attached.This thread cannot be a system thread. A thread cannot attach to itself. Therefore, idAttachTo cannot equal idAttach.</param>
+        /// <param name="fAttach">If this parameter is TRUE, the two threads are attached.If the parameter is FALSE, the threads are detached.</param>
+        /// <returns>If the function succeeds, the return value is nonzero. If the function fails, the return value is zero.To get extended error information, call GetLastError. Windows Server2003 and WindowsXP:  There is no extended error information; do not call GetLastError.This behavior changed as of WindowsVista.</returns>
```

Comment:
You are missing several spaces between words and sentences. Likely this happened when you copied and pasted the MSDN docs and C# "formatted" the text by removing spaces after periods. It's important when you copy and paste that after paste, you immediately hit Undo to revert the formatting changes (but keep the paste).